Javacalc/Readme.md

```diff
@@ -0,0 +1,6 @@
+Instruction Java Calculator
+
+1) cd ../.../your directory with that calculator
+2) $ javac calculator.java
```

Comment:
OK, it is working, but only because OS X filesystem is not case-sensitive by default:
```
GDSpb0659:Javacalc dmyasnikov$ javac calculator.java
```